nntrainer/tensor/tensor.cpp

```diff
@@ -889,10 +872,35 @@ void Tensor::reshape(const TensorDim &d) {
   if (d.getDataLen() != dim.getDataLen()) {
     throw std::invalid_argument("Error: reshape cannot change the tensor size");
   }
+
   dim = d;
   strides = d.computeStrides();
 }
 
+void Tensor::fill(const Tensor &from, bool initialize) {
+  if (initialize && this->uninitialized()) {
+    return this->copy(from);
```

Comment:
return in a function which returns void?

Reply:
Updated